src/ui/server/controllers/insurance/your-buyer/map-submitted-data/buyer-trading-history/index.test.ts

```diff
@@ -46,4 +49,56 @@ describe('controllers/insurance/your-buyer/map-submitted-data/buyer-trading-hist
       expect(response).toEqual(expected);
     });
   });
+
+  describe(`when ${OUTSTANDING_PAYMENTS} is set to "false"`, () => {
+    it(`should return mockFormBody without _csrf and ${TOTAL_OUTSTANDING_PAYMENTS} and ${TOTAL_OVERDUE_PAYMENTS} set to "null"`, () => {
+      mockFormBody[OUTSTANDING_PAYMENTS] = 'false';
+
+      const response = mapSubmittedData(mockFormBody);
+
+      const { _csrf, alternativeCurrencyCode, ...expectedBody } = mockFormBody;
+
+      const expected = {
+        ...expectedBody,
+        [TOTAL_OUTSTANDING_PAYMENTS]: null,
+        [TOTAL_OVERDUE_PAYMENTS]: null,
+      };
+
+      expect(response).toEqual(expected);
+    });
+  });
+
+  describe(`when ${TOTAL_OUTSTANDING_PAYMENTS} is set to an empty string`, () => {
+    it(`should return mockFormBody without _csrf and ${TOTAL_OUTSTANDING_PAYMENTS} set to "null"`, () => {
+      mockFormBody[TOTAL_OUTSTANDING_PAYMENTS] = '';
+
+      const response = mapSubmittedData(mockFormBody);
+
+      const { _csrf, alternativeCurrencyCode, ...expectedBody } = mockFormBody;
+
+      const expected = {
+        ...expectedBody,
+        [TOTAL_OUTSTANDING_PAYMENTS]: null,
+      };
+
+      expect(response).toEqual(expected);
+    });
+  });
+
+  describe(`when ${TOTAL_OVERDUE_PAYMENTS} is set to an empty string`, () => {
+    it(`should return mockFormBody without _csrf and ${TOTAL_OVERDUE_PAYMENTS} set to "null"`, () => {
+      mockFormBody[TOTAL_OVERDUE_PAYMENTS] = '';
+
+      const response = mapSubmittedData(mockFormBody);
```

Comment:
Noticed this in a few places, have been renaming, can we use `result` instead of `response`?